sqlx-core/src/executor.rs

```diff
@@ -172,7 +172,7 @@ pub trait Execute<'q, DB: Database>: Send + Sized {
     /// will be prepared (and cached) before execution.
     fn take_arguments(&mut self) -> Option<<DB as HasArguments<'q>>::Arguments>;
 
-    /// Returns true if query has any parameters.
```

Comment:
Note. This was never on master, but on this branch as an idea (that was never used in the end).